cpp/src/neighbors/detail/cagra/search_multi_cta_kernel.cuh

```diff
@@ -15,7 +15,7 @@
  */
 #pragma once
 
-#ifndef CUVS_EXPLICIT_INSTANTIATE_ONLY
+#ifndef _CUVS_EXPLICIT_INSTANTIATE_ONLY

```

Comment:
Hi @cjnolet , I'm not so sure if our Cagra module needs to open this macro, so add a `_` for the placeholder. If you feel it is necessary, could you help tag a suitable reviewer to take a look? Thanks!